models/webhooks/store_modifier_updated.yml

```
@@ -0,0 +1,42 @@
+type: object
+properties:
+  store/modifier/updated:
+    description: |
+      Fires when you edit attributes for a local or shared modifier. Updates to the global store and channel locale as overrides triggers the webhook. The `context` fields are present for only override updates. 
+      
+      For shared modifiers, the maximum number of product IDs in a payload is 100. If a shared modifier is assigned to more than 100 products, the payload includes a link to the API so you can query all the event data.  
+    type: object
+    properties:
+      scope:
+        type: string
+      store_id:
+        type: string
+      data:
+        type: object
+        properties:
+          type:
+            type: string
+            enum:
+            - local_modifier
+            - shared_modifier
+          id:
+            type: integer
+          affected_product_ids: 
+            type: array
+            items:
+              type: string
```

```suggestion
              type: string
```
What's the type? Is it an array of strings (like what I have documented) or an array of integers?

---

Should be integers

---

Thanks, fixed